integration-libs/cpq-quote/cpq-quote-discount/cpq-qute.service.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// cpq-quote-status.service.ts
+import { Injectable } from '@angular/core';
+import { CartItemListComponentService } from '@spartacus/cart/base/components';
+import { BehaviorSubject } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CpqQuoteService extends CartItemListComponentService {
+  private isFlagSubject = new BehaviorSubject<boolean>(true); // Default value is true
+  isFlag$ = this.isFlagSubject.asObservable();
+
+  setIsFlag(value: boolean) {
+    this.isFlagSubject.next(value);
+  }
+  showBasePriceWithDiscount(): boolean {
```

Comment:
@Jali-Khusi, please, add a test for this method: 
![image](https://github.com/SAP/spartacus/assets/61147963/2ed187a2-fc65-4111-8f32-c951a04915bc)